src/i18n/data.json

```diff
@@ -71,6 +74,9 @@
         "rule.disableImageMessages.title": "Ei kuvaviestejä",
         "rule.inputMinHeight.title": "Viestikentän korkeus",
         "rule.inputMinHeight.description": "Aseta viestien kirjoituskentän korkeus (väliltä 1 ja 25).",
-        "rule.disableInputMinHeight.title": "Palauta alkuperäinen viestikentän korkeus"
+        "rule.disableInputMinHeight.title": "Palauta alkuperäinen viestikentän korkeus",
+        "rule.noEmojis.title": "Kiellä emojit",
```

Comment:
I think we should talk about "hymiö" instead of "emoji", even though emoji is the correct term. "Emoji" is rarely used in finnish speech. (Mainly an opinion, so not a blocker).